### ISSUE TITLE
Running make deletes committed test file

### ISSUE THREAD

**Issue by jfkw** [millejoh/emacs-ipython-notebook#60]
Running make will delete a committed file tests/notebook/Islands Tests.ipynb. Would it be possible to replace this file at the end of the test, or test deletion using a copy? The benefit would be keeping a clean git checkout.
```
% make
(...)
% git status
 D "tests/notebook/Islands Tests.ipynb"
?? tests/notebook/Untitled0.ipynb
?? tests/notebook/Untitled1.ipynb
?? tests/notebook/Untitled2.ipynb
?? tests/notebook/Untitled3.ipynb
?? tests/notebook/Untitled4.ipynb
?? tests/notebook/Untitled5.ipynb
?? tests/notebook/Untitled6.ipynb
```

**Comment by millejoh** [comment]
That test file isn't used for anything, so I have deleted it. Almost all of my development work is on Windows, so I rarely use make. I tried committing a small change to delete the ipynb files when calling `make clean`. Let me know if that is the behavior you are hoping for.

**Comment by jfkw** [comment]
Fixed with 10044ddc9d3f93ac8f423161a600825e26e3d21a
